libc-test/build.rs

```diff
@@ -289,6 +298,24 @@ fn main() {
             "FILE_ATTRIBUTE_INTEGRITY_STREAM" |
             "ERROR_NOTHING_TO_TERMINATE" if mingw => true,
 
+            // these are not in musl
+            "SOL_NETLINK" |
+            "SOL_UDP" |
+            "SOL_AX25" |
+            "SOL_ATALK" |
+            "SOL_NETROM" |
+            "SOL_IPX" |
+            "SOL_ROSE" if musl => true,
+
+            // these are not in glibc or musl yet (but they are in android)
+            "SOL_SCTP" |
+            "SOL_UDPLITE" |
+            "SOL_NETBEUI" |
+            "SOL_LLC" |
+            "SOL_DCCP" |
+            "SOL_NETLINK" |
+            "SOL_TIPC" if linux => true,
```

Comment:
Hm I'm a little wary of including constants that aren't in glibc or musl yet. Perhaps these could be android-only for now? Or should we update glibc to start verifying them?